### ISSUE TITLE
Banner: Remove the alt text description

### ISSUE THREAD

**Issue by jAigret-Bix** [uswds/uswds#3689]
USAGov accessibility test results: Banner
The alt text defined for the government building and lock images is incorrect.  
Equivalent alternate text is needed for all the images in order to convey the same level of information as the image itself for users who cannot see the images. 
Update the alt text to describe what is being conveyed in the image. For the .gov image add “government building” and for the lock image add “lock” alt text.

[Test results](https://docs.google.com/document/d/1g5beRCFgb5M7xYrEpBPuZy39AKYxtCXzT54FHTPiRgg/edit)

![Banner](https://user-images.githubusercontent.com/67592482/94595720-31749180-0259-11eb-9c83-8fd5476b0cab.png)

- [x] Define: if it's supposed to have an ALT then we need to write more descriptive text. If it's decorative, remove the value within the alt quotes (e..g., alt=“”) so that it’s treated as decorative and not a poorly described non-decorative image.
- [x] Direction is that the image is decorative. Removed the alt text description.

**Comment by jAigret-Bix** [comment]
Required direction from @thisisdano. If it's supposed to have an ALT then we need to write more descriptive text. If it's decorative, then this is a non-issue and can be closed.
